index.js

```diff
@@ -0,0 +1,107 @@
+document.addEventListener("DOMContentLoaded", () => {
+    let form1 = document.querySelector("#form1");
+    let form2 = document.querySelector("#form2");
+    let form3 = document.querySelector("#form3");
+    let addClassName = document.querySelector("#class");
+    let addTeacher = document.querySelector("#teacher");
+    let studentClass = document.querySelector("#studentClass");
+    let studentName = document.querySelector("#studentName");
+    let studentAge = document.querySelector("#studentAge");
+    let studentCity = document.querySelector("#studentCity");
+    let studentGrade = document.querySelector("#studentGrade");
+    let passingClass = document.querySelector("#passingClass");
+    let City = document.querySelector("#city");
+    let failing = document.querySelector("#failing");
+    let ul = document.querySelector("ul")
+
+    const addClass = (className, teacher) => {
+        ul.innerHTML = "";
+        axios.post(`http://localhost:3000/class`, {
```

Comment:
You should assign this to a variable and use the return value of the post request to display the info to the user